screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionScreen.kt

```diff
@@ -56,7 +56,7 @@ import com.ivy.navigation.navigation
 import com.ivy.navigation.screenScopedViewModel
 import com.ivy.ui.R
 import com.ivy.wallet.domain.data.CustomExchangeRateState
-import com.ivy.wallet.domain.data.IvyCurrency
+import com.ivy.legacy.domain.data.IvyCurrency
```

Comment:
What caused this package change?

Reply:
Package directive of `IvyCurrency` data class  was wrong.  It resides in `com.ivy.legacy.domain.data.IvyCurrency` but directive was `com.ivy.wallet.domain.data.IvyCurrency` in the data class file. I fixed it thus these changes appeared as-per correct package directive in places where it's used.